frontend/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "frontend"
+build = "build.rs"
+version.workspace = true
+authors.workspace = true
+license-file.workspace = true
+edition.workspace = true
+readme.workspace = true
+repository.workspace = true
+
+[lib]
+name = "frontend"
+path = "build.rs"
```

Comment:
It is better to create dummy `frontend.rs` file. With the current approach `cargo` generates warning:
```
warning: \frontend\Cargo.toml: file `\frontend\build.rs` found to be present in multiple build targets:
  * `lib` target `frontend`
  * `build-script` target `build-script-build`
```

Reply:
feels clunky to generate empty files **just** to eliminate warning

Reply:
Interestingly enough, setting the path to `/dev/null`  works fine on unix. Need to test that `NUL` on windows works too